s3/pkg/meta/db/drivers/tidb/bucket.go

```diff
@@ -229,7 +229,7 @@ func (t *TidbClient) PutBucket(ctx context.Context, bucket *Bucket) error {
 	lc, _ := json.Marshal(bucket.LifecycleConfiguration)
 	bucket_policy, _ := json.Marshal(bucket.BucketPolicy)
 	sql := "update buckets set bucketname=?,acl=?,policy=?,cors=?,lc=?,tenantid=?,versioning=? where bucketname=?"
-	args := []interface{}{bucket.Name, acl, bucket_policy, cors, lc, bucket.TenantId, bucket.Versioning, bucket.Name}
+	args := []interface{}{bucket.Name, acl, bucket_policy, cors, lc, bucket.TenantId, bucket.Versioning.Status, bucket.Name}
```

Comment:
We fixed this part earlier, right?

Reply:
This is another one, the one we fixed 3 days ago are for creating bucket, this one is for updating bucket.